src/plugins/auto/plugin.cpp

```diff
@@ -503,15 +503,19 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         const auto& deviceConfig = p.config;
         SoExecutableNetworkInternal exec_net;
         LOG_DEBUG_TAG("load network to device:%s", deviceName.c_str());
-        if (modelPath.empty()) {
-            exec_net = GetCore()->LoadNetwork(network, deviceName, deviceConfig);
-        } else if (GetCore()->DeviceSupportsImportExport(deviceName)) {
-            exec_net = GetCore()->LoadNetwork(modelPath, deviceName, deviceConfig);
-        } else {
-            std::call_once(readNetworkFlag, [&]() {
-                network = GetCore()->ReadNetwork(modelPath, std::string());
-            });
-            exec_net = GetCore()->LoadNetwork(network, deviceName, deviceConfig);
+        try {
+            if (modelPath.empty()) {
+                exec_net = GetCore()->LoadNetwork(network, deviceName, deviceConfig);
+            } else {
+                exec_net = GetCore()->LoadNetwork(modelPath, deviceName, deviceConfig);
+            }
+        } catch (const IE::Exception&) {
+            if (_LogTag == "AUTO") {
+                LOG_DEBUG_TAG("Failed to load network to device:%s", deviceName.c_str());
+                return;
+            } else {
+                IE_THROW() << "Failed to load network to device: " << deviceName.c_str();
+            }
         }
```

Comment:
Also, please enable mock tests to test the new logic

Reply:
Okay

Reply:
Added the test case.